experiment/simulation/js/fa.js

```diff
@@ -0,0 +1,333 @@
+import { registerGate } from "./main.js";
+import { setPosition } from "./layout.js";
+import {gates} from './gate.js';
+import {jsPlumbInstance} from "./main.js";
+
+// Dictionary of all full adders in the circuit with their IDs as keys
+let fullAdder = {}
+
+function clearFAs() {
+
+    for (let faID in fullAdder) {
+        delete fullAdder[faID];
+    }
+    fullAdder = {};
+}
+
+// {output-id: [gate,pos]}
+let finalOutputs = {

```

Comment:
Can this be made const?

Reply:
yes it can be made const.I have updated the file